Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Custom Unions

## Changelog:
- Added an option for the players to **_Found a New Union_** in the decisions. There can be only one custom union. The option should be around year 603. Rulers of the same cultural group and religion prefer to join such unions.
- Added the Features tooltip to the Unions tab explaining some union features.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/111083440-16f05500-8527-11eb-925e-abfc6999d0be.png)

</details>

# Tasks:
- [x] Test every feature again
- [x] Look for horde, alliance, union errors
- [x] Add the event where you enter a custom union name
- [x] Add custom union description
- [x] Add custom union features
- [x] Change decision pictures


# How to test:
Create a custom union and check for basic functions like joining, leaving, renaming, inviting.

[Filter](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/233/files?file-filters%5B%5D=.yml) to hide non-localization files for @Warcraft-GoA-Development-Team/english-proofreaders.
